themes/default/content/blog/reducing-our-error-rate/index.md

```diff
@@ -0,0 +1,74 @@
+
+If you want to be able to read every error message, then the error rate has to come down as API traffic increases.
+
+## Why Should You Care?
+
+Pulumi aspires to be the most reliable infrastructure that our customers interact with, and the benefits towards that end are reason enough for us. But this process is by no means free, and there is always an opportunity cost.
+
+We noticed a powerful second-order effect emerge over time. The team began obsessing over the user experience. Following this process builds a visceral understanding of how your system behaves. You know when a new feature has a bug before the first support tickets get opened. You know which customer workloads will require scaling investments in the next 90 days. You know what features see heavy usage and which ones customers ignore. You’re forced to confront every wart in your application. Slowly, your team builds a better understanding of your customers and this trickles down into every aspect of product development. You begin to hold yourselves to higher standards.
+
+A friend and engineer at a large-cap software company read a draft of this post and told me:
+
+ > “I can’t imagine this process being set on any of the systems I work on at *REDACTED*. They are so prone to 500s and everybody shrugs their shoulders as if it is just commonplace.”
+
+Apathy in place of customer obsession is not an option if you are a startup that wants to disrupt anything.
+
+## The SRE's Folly
+
+Error budgets and the SRE model are haute couture. Some preach that we should never look at errors at this level of granularity and instead use expensive tools that aggregate, categorize, and collect statistics on errors flowing through your system. But all of this automation can actually make things worse when you reach for it prematurely. Aggregating errors is a great way to gloss over important details early on. Collecting fancy metrics does not matter if your users are not happy. Cutting your teeth with the tools and processes that make sense for your level of scale is the only way to build a high-performance culture. Skipping straight to step 100 does not always help.
+
+Admittedly, this process does not work for Google-level scale. But it works a lot longer than you might imagine. Pulumi manages a large fraction of the resources deployed across all clouds. I asked large-cap software engineer about traffic levels and believe it or not, it is in the same order of magnitude as what we see at Pulumi.
```

Comment:
I feel like we should re-phrase "a large fraction of the resources deployed across all clouds."  But don't have a clear suggestion.

Reply:
What is your goal for the restatement? Do you feel like this over or undersells? 

Alternatives could be:
- a meaningful fraction
- a meaningful percentage